src/main/java/com/example/onlinebookstore/repository/BookRepository.java

```diff
@@ -1,10 +1,7 @@
 package com.example.onlinebookstore.repository;
 
 import com.example.onlinebookstore.model.Book;
-import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
 
-public interface BookRepository {
-    Book save(Book book);
-
-    List<Book> findAll();
+public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
I believe you should add the repository in the different task. Why is it present in this PR? :)